dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -116,3 +116,22 @@ object reflects the newly requested size in `PersistentVolume.Spec.Capacity`. At
 this point, you can create or re-create a new pod from the PVC to finish the
 file system resizing. Once the pod is running, the newly requested size is
 available and `FileSystemResizePending` condition is removed from the PVC.
+
+[[recover_from_resize_failure]]
+== Recovering from Failure when Expanding Volumes
+
+If expanding underlying storage fails either on master or node,
+{product-title} admin can manually recover the PVC state and cancel the resize requests that are
+continuously retried by the controller without administrator intervention.
+
+Currently, this can be done manually by completing the following steps:
+
+1. Mark the PV that is bound to the claim (PVC) with the `Retain` reclaim policy.
+   This can be done by editing the PV and changing persistentVolumeReclaimPolicy to `Retain`.
+2. Delete the PVC, so as we can re-create it later.
+3. To ensure that newly created PVC can bind to PV marked `Retain`, we will have to manually edit the PV
+   and delete `claimRef` entry from PV specs. This will mark the PV as `Available`. For more information
+   about prebinding PVCs please see xref:./persistent_volumes.adoc#persistent-volumes-volumes-and-claim-prebinding[Volume and claim prebinding].
+4. Re-create the PVC in a smaller size or a size that can be allocated by the underlying storage provider.
```

Comment:
s/or a size that can be allocated by the underlying storage provider./that the underlying storage provider can allocate.

Reply:
That is not quite accurate, sometimes an admin may just want to cancel the operation and hence we will want to restore the size to original value so as resizing operation will be attempted no more.

Reply:
OK, sounds good. Thanks.